rules_doc_generator/model/main.py

```diff
@@ -13,7 +13,7 @@ def create_toc_html(id_map: RefDict):
   return result
 
 def standalone_html(document: Document, config: Config, id_map: RefDict):
-  result = '<!DOCTYPE html><html><head><link rel="stylesheet" href="rules.css"><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Atkinson+Hyperlegible"></head><body>'
+  result = '<?xml encoding="utf-8" ?><!DOCTYPE html><html><head><script src="html/rules.js" defer></script><link rel="stylesheet" href="html/rules.css"><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Atkinson+Hyperlegible"><link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" /></head><body>'
```

Comment:
Now there's a mismatch for the static html site, as the css is beside the index.html there. I think maybe the easiest way to fix it is just to follow a similar pattern for the `php` site: add the `index.php` to the templates and copy everything into an output `php` folder how it should be for the `php` site.

Reply:
Cleaned all of this up. Now the HTML output is slightly adjusted depending on whether we output for the html stand-alone or the php server version.